EIPS/eip-1052.md

```diff
@@ -18,16 +18,57 @@ Many contracts need to perform checks on a contract's bytecode, but do not neces
 Contracts can presently do this using the `EXTCODECOPY` opcode, but this is expensive, especially for large contracts, in cases where only the hash is required. As a result, we propose a new opcode, `EXTCODEHASH`, which returns the keccak256 hash of a contract's bytecode.
 
 ## Specification
-A new opcode, `EXTCODEHASH`, is introduced, with number 0x3D. `EXTCODEHASH` takes one argument from the stack, and pushes the keccak256 hash of the code at that address to the stack.
+
+A new opcode, `EXTCODEHASH`, is introduced, with number 0x3D. The `EXTCODEHASH` 
+takes one argument from the stack, zeros the first 96 bits 
+and pushes to the stack the keccak256 hash of the code of the account 
+at the address being the remaining 160 bits. 
+
+In case the account does not exist `0` is pushed to the stack.
```

Comment:
```suggestion
In case the account does not exist or is empty `0` is pushed to the stack.
```
AFAIK there are still _some_ empty accounts (the vast majority of empty accounts were cleared during the "state clearing process" but only the ones the attacker created were specifically targeted). Thus it is necessary to specify the behavior.